src/Microsoft.VisualStudio.Web.CodeGeneration.Contracts/Microsoft.VisualStudio.Web.CodeGeneration.Contracts.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\..\build\common.props" />
   <PropertyGroup>
-    <TargetFramework>netstandard1.3</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
Why does this have to move to `netcoreapp2.0`
Shouldn't the libraries remain at `netstandard`?